packages/web/pages/_document.tsx

```diff
@@ -24,10 +24,6 @@ class MetaDocument extends Document {
             rel="stylesheet"
           />
           <meta charSet="UTF-8" />
-          <meta
-            name="viewport"
-            content="width=device-width, initial-scale=1.0"
-          />
```

Comment:
This removes the following warning:
Warning: viewport meta tags should not be used in _document.js's <Head>. 
https://err.sh/next.js/no-document-viewport-meta